packages/nimble-components/src/rich-text/models/tests/markdown-parser.spec.ts

```diff
@@ -234,6 +234,100 @@ describe('Markdown parser', () => {
         });
     });
 
+    describe('escape backslashes should be ignored while parsing', () => {
```

Comment:
This looks like good coverage of invalid escape characters being ignored. But do we have coverage of valid characters being handled correctly? For example, that `\n` results in a line break or `\\` results in a backslash.

Reply:
We don't have a test separately to check for proper rendering of `\n`, `\\`. Will add them.